app/tests/spec/views/cannot_create_account.js

```diff
@@ -52,5 +57,27 @@ function (chai, sinon, View, Relier) {
           assert.equal(view.$('.ftc').attr('target'), null);
         });
     });
+
+    it('has a working `Learn More` link with the default broker', function () {
+      return view.render()
+        .then(function () {
+          assert.isFalse(view.$('.show-visible-url').length > 0);
+        });
+    });
+
+    it('has a `Learn More` link converted to text with `convertExternalLinksToText` capability', function () {
+      sinon.stub(broker, 'hasCapability', function (capability) {
+        if (capability === 'convertExternalLinksToText') {
+          return true;
+        }
+
+        return false;
+      });
+
+      return view.render()
+        .then(function () {
+          assert.isTrue(view.$('.show-visible-url').length > 0);
```

Comment:
I just learned about chai's [isAbove](http://chaijs.com/api/assert/#isAbove-section), this would be a good place to use it.